### PR TITLE
fix create BLB failed issue

### PR DESCRIPTION
**Pull request by hello2mao**
This PR fix issue #13.

### CODE REVIEW

**Comment by hello2mao** [comment]
@m3ngyang hi, please help review this pr.

File: pkg/cloud-provider/load_balancer.go [hunk 1]
```diff
@@ -793,3 +796,71 @@ func (bc *BCECloud) waitForLoadBalancer(lb *blb.LoadBalancer) (blb.LoadBalancer,
...
+	params := make(map[string]string, 0)
+	params["vpcId"] = subnet.VpcID
+	subnets, err := bc.clientSet.Vpc().ListSubnet(params)
+	if err != nil {
+		return "", "", fmt.Errorf("ListSubnet failed: %v", err)
+	}
+	for _, subnet := range subnets {
+		if subnet.Name == "系统预定义子网" {
+			return subnet.VpcID, subnet.SubnetID, nil
+		}
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go:831]:
```golang
else if subnet.Name == "CCE-Reserve" {
}
```

File: pkg/cloud-provider/load_balancer.go [hunk 2]
```diff
@@ -793,3 +796,71 @@ func (bc *BCECloud) waitForLoadBalancer(lb *blb.LoadBalancer) (blb.LoadBalancer,
 
 	return *lb, nil
 }
+
+func (bc *BCECloud) getVpcInfoForBLB() (string, string, error) {
+	// get prefer vpc info
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go:801]: perfer -> perferred

File: pkg/cloud-provider/load_balancer.go [hunk 3]
```diff
@@ -793,3 +796,71 @@ func (bc *BCECloud) waitForLoadBalancer(lb *blb.LoadBalancer) (blb.LoadBalancer,
...
+	// check subnet
+	subnet, err := bc.clientSet.Vpc().DescribeSubnet(subnetId)
+	if err != nil {
+		return "", "", fmt.Errorf("DescribeSubnet failed: %v", err)
+	}
+	if subnet.SubnetType == "BCC" {
+		return vpcId, subnetId, nil
+	}
+
+	// get subnet list and choose prefer one
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go]: prefer -> perferred

File: pkg/cloud-provider/load_balancer.go [hunk 4]
```diff
@@ -793,3 +796,71 @@ func (bc *BCECloud) waitForLoadBalancer(lb *blb.LoadBalancer) (blb.LoadBalancer,
...
+			Name:       "CCE-Reserve",
+			ZoneName:   subnet.ZoneName,
+			Cidr:       nextCidr.String(),
+			VpcID:      subnet.VpcID,
+			SubnetType: "BCC",
+		}
+		newSubnetId, err := bc.clientSet.Vpc().CreateSubnet(createSubnetArgs)
+		if err != nil {
+			glog.V(4).Infof("CreateSubnet failed: %v, will try again.", err)
+			continue
```
Comment (m3ngyang) [inline pkg/cloud-provider/load_balancer.go:861]: sleep for some time before trying again

Comment (hello2mao) [reply pkg/cloud-provider/load_balancer.go:861]: ok, I'll add 3s sleep.

**Review by m3ngyang — APPROVED** [review]
LGTM

**Pull request merged**